hawc/apps/assessment/api.py

```diff
@@ -165,6 +165,14 @@ class Assessment(AssessmentViewset):
     serializer_class = serializers.AssessmentSerializer
 
 
+class PublicAssessment(viewsets.ReadOnlyModelViewSet):
+    model = models.Assessment
+    serializer_class = serializers.PublicAssessmentSerializer
+
+    def get_queryset(self):
+        return self.model.objects.get_public_assessments()
```

Comment:
we can add the `get_queryset` method here into the new action view we define above